.github/workflows/deploy-terraform-to-gcp.yml

```
@@ -0,0 +1,108 @@
+  GCP_REGION: europe-west6
+  ARTIFACT_REGISTRY_NAME: staging-website-google-artifact-registry-repository
+  TERRAFORM_DIR: website/infra
+
+jobs:
+  deploy:
+    name: Staging
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 📦 Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)